lib/klp.pm

```diff
@@ -70,7 +79,16 @@ sub is_klp_pkg {
                 kver => $kver,
                 kflavor => $kflavor,
             };
-
+        } elsif ($$pkg{name} =~ m/^${base}-(\d+_\d+_\d+-\d+_stage_\d*)-([a-z][a-z0-9]*)$/) {
```

Comment:
This regex could be merged into the original one using `...-\d+(?:_stage_|_*\d*)_*\d*...`. The only other change would be the `$kver` underscore replacement.